packages/desktop-client/src/browser-preload.browser.js

```diff
@@ -119,12 +119,12 @@ global.Actual = {
 document.addEventListener('keydown', e => {
   if (e.metaKey || e.ctrlKey) {
     // Cmd/Ctrl+o
-    if (e.key === 'O') {
+    if (e.key === 'o') {
       e.preventDefault();
       window.__actionsForMenu.closeBudget();
     }
     // Cmd/Ctrl+z
-    else if (e.key === 'Z') {
+    else if (e.key === 'z' || e.key === 'Z') {
```

Comment:
```suggestion
    else if (e.key.toLowerCase() === 'z') {
```

And same for the others. WDYT?

Reply:
For the others we explicitly don’t want to handle the capitalized version IMO